teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -9,7 +9,8 @@
     "Authors:\n",
```

Comment:
<p>Use <code>.head()</code>  so we get 5 compounds instead. Two is too little :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p><code>.head(5)</code>  may make the output too long?  Maybe show 3 compounds?</p>

Reply:
<p>done</p>